runtime/tests/runtime_integration_tests/test_get_idev_csr.rs

```diff
@@ -0,0 +1,61 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_api::SocManager;
+use caliptra_common::mailbox_api::{CommandId, GetIdevIdCsrResp, MailboxReqHeader};
+use caliptra_drivers::{IDevIDCsr, MfgFlags};
+use caliptra_error::CaliptraError;
+use caliptra_hw_model::{HwModel, ModelError};
+use caliptra_runtime::RtBootStatus;
+use openssl::x509::X509Req;
+use zerocopy::{AsBytes, FromBytes};
+
+use crate::common::{run_rt_test, RuntimeTestArgs};
+
+#[test]
+fn test_get_csr() {
+    let args = RuntimeTestArgs {
+        test_mfg_flags: Some(MfgFlags::GENERATE_IDEVID_CSR),
```

Comment:
nit: might be worth a comment that `run_rt_test` will clear this flag to allow RT to boot.